src/web/templates/user_boards_screen_set.html

```diff
@@ -14,9 +20,9 @@ <h2 class="set-title text-center">{{ screen_set.name_for_boards }} (ronde {{ tou
                                         <th scope="col" class="board-number">Ech</th>
                                         <th scope="col" class="points">Pts</th>
                                         {% if tournament.print_real_points %}<th scope="col" class="points">&nbsp;</th>{% endif %}
-                                        <th scope="col" class="player w-50" {% if screen.type == 'input' and tournament.record_illegal_moves %}colspan="2">{% endif %}Blancs{% if tournament.handicap %} (temps){% endif %}</th>
+                                        <th scope="col" class="player w-50" {% if screen.type == 'input' and tournament.record_illegal_moves %}colspan="2">{% endif %}{{ _('White') }}{% if tournament.handicap %} ({{ _('time control') }}){% endif %}</th>
                                         <th scope="col" class="score">Score</th>
-                                        <th scope="col" class="player w-50" {% if screen.type == 'input' and tournament.record_illegal_moves %}colspan="2">{% endif %}Noirs{% if tournament.handicap %} (temps){% endif %}</th>
+                                        <th scope="col" class="player w-50" {% if screen.type == 'input' and tournament.record_illegal_moves %}colspan="2">{% endif %}{{ _('Black') }}{% if tournament.handicap %} ({{ _('time control') }}){% endif %}</th>
```

Comment:
I would prefer if both white and black were visible on all input or boards screen set